plugin.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+Object.defineProperty(exports, '__esModule', {
+  value: true,
+});
+exports.cachePath = undefined;
+
+var _extends =
+  Object.assign ||
+  function(target) {
+    for (var i = 1; i < arguments.length; i++) {
+      var source = arguments[i];
+      for (var key in source) {
+        if (Object.prototype.hasOwnProperty.call(source, key)) {
+          target[key] = source[key];
+        }
+      }
+    }
+    return target;
+  };
+
+exports.matchImageSnapshotOptions = matchImageSnapshotOptions;
+exports.matchImageSnapshotResult = matchImageSnapshotResult;
+exports.matchImageSnapshotPlugin = matchImageSnapshotPlugin;
+exports.addMatchImageSnapshotPlugin = addMatchImageSnapshotPlugin;
+
+var _fsExtra = require('fs-extra');
```

Comment:
why not const? also `_` means private for me, and this is not a private variable. I would `const fsExtra = require('fs-extra');`

Reply:
also, other files in the project use `import` syntax. plz support if possible.
ref: https://github.com/palmerhq/cypress-image-snapshot/blob/master/src/plugin.js

Reply:
> why not const? also `_` means private for me, and this is not a private variable. I would `const fsExtra = require('fs-extra');`

this is the same way of coding in the same file